### DISCUSSION TITLE
Clarify acceptable cases of "conflict of interest"

### DISCUSSION THREAD

**Discussion by Sanne** [commonhaus/foundation#58] [General]
The document about [Conflict of Interest](https://github.com/commonhaus/foundation-draft/blob/main/policies/conflict-of-interest.md) currently states:

>   A conflict of interest occurs when an individual's personal, professional, or financial interests could potentially interfere with or influence their decisions or actions on behalf of the CF.

And specifically makes the examples:

> **Financial interest** in any entity with which CF transacts. Board membership, **employment**, or volunteer work with such an entity.

I would expect it to be fairly common that many of us will be employed by entities which also sponsor CF in other ways, or merely by employing ourselves for the purpose of contributing to projects; this is also likely to result in other forms of financial interest, e.g. doing employer doing well financially.

This might need to be clarified especially as I'd expect it to be a common case?

**Comment by ebullient** [comment]
Yes. This overlaps with a point that was raised elsewhere, but where I land is this: if you (as an individual) must vote in a certain way (because of your employment contract, for example), but the decision we're considering benefits your company specifically or solely, then you'd want to abstain. 

So, yes.. most will have an employer, and to some extent (according to contract), will be required to play in alignment with their employer's interest. Most of the time, I don't think this will be a problem: project goals, foundation goals, and employer goals will all be in alignment and will keep projects moving forward.

When there is conflict or disagreement that centers on an issue that disproportionately benefits a single sponsoring organization, that's where we have a problem, and that comes down to the 501(c)(6) rules about benefit to a single organization or individual.

**Reply by Sanne** [reply]
This is really tricky, imagine this hypotethical scenario: a strong disagreement happens between two members of the same project, they vote for opposite solutions, both honestly and in good faith thinking it's the best tradeoff for the sake of the future of the project; obviously they come from different perspectives and experiences, possibly influenced by their use & needs of the project so it might easily look like each one is voting as to defend their company's use case and bias. Will they both start accusing each other of having breached these rules?

> When there is conflict or disagreement that centers on an issue that disproportionately benefits a single sponsoring organization

I quite like this attempt, but the term "disproportionately" is both key and fairly subjective... seems very tricky to me as rather than being of guidance for a conflict of resolution which could have been moderately bad, it could make it even worse so as to escalate beyond the technical issue to also become a breach of conduct.

How bad would it be if there was an accepted degree of "conflict of interest is unavoidable" ?

**Reply by ebullient** [reply]
"conflict of interest is unavoidable" -- that is truth. 

> obviously they come from different perspectives and experiences, possibly influenced by their use & needs of the project so it might easily look like each one is voting as to defend their company's use case and bias

The key here is transparency and disclosure. 

If there is a conflict of interest, others (the CFC and EGC should have representation from other companies, e.g.) can then help rationalize, remediate, compromise, etc. If all the cards are on the table, we can find a way out. That does require a willingness to "behave" (be willing to resolve conflict constructively), which I guess leads to expectations re: code of conduct.

**Comment by gavinking** [comment]
I think this part of the current language here is a bit confusing, and maybe too heavy-handed:

> Upon the disclosure of a potential conflict, the individual involved must abstain from participating in discussions or decision-making processes related to the matter. The CFC will review the situation and determine the appropriate course of action.

- This sort of suggests that the intent here is that the CFC may decide that the disclosed conflict of interest is benign, and that the individual may resume participation. But it doesn't actually say that.
- It seems to me that disclosure => immediate abstention even from "participating in discussions" goes too far. (And even creates a disincentive to disclosure.)

In general I would say that the disclosure part of this is actually more important than abstention. Some "potential conflicts" of interest indeed require abstention, but is it really the case that abstention would always be required by default?

That said, I'm not sure what is the norm here in other nonprofits, so I'm kinda just talking out my ass. Perhaps abstention by default is correct.

**Reply by Sanne** [reply]
Right I think this better describes my concern: if we all have, always and unavoidably, some degree of conflict of interest (as mentioned and apparently agreed above) then it shouldn't be expected that everyone abstains from any discussion in such cases as that would invalidate any and all discussions.

The fact that this creates a disincentive to disclosure is also a red flag.

I'd rather try to promote and encourage an habit of full-transparency, where such practice never results in any form of retaliation or limitation? Then others can take such information into account when weighing any proposal, even though I'd hope proposals would be evaluated based on their merit alone.

**Reply by ebullient** [reply]
Disclosure should be required, at a minimum. Limitation of "participation in discussion" to clarify it applies to "when making the final decision" may make a difference. Here are some references (and I'll do some double checking also to make sure language is correct):

- https://www.irs.gov/charities-non-profits/form-1023-purpose-of-conflict-of-interest-policy
- https://www.councilofnonprofits.org/running-nonprofit/governance-leadership/conflicts-interest
- https://nonprofitquarterly.org/charity-conflicts-of-interest-a-guide/
- https://www.nolo.com/legal-encyclopedia/conflict-of-interest-policies-for-nonprofit.html

Tax exempt status is on the line. There are real consequences for "self-dealing" (decisions that benefit a single company or individual).

**Reply by gavinking** [reply]
So for example, that second link says:

> A policy on conflicts of interest should (a) require those with a conflict (or who think they may have a conflict) to disclose the conflict/potential conflict, and (b) prohibit interested board members from voting on any matter in which there is a conflict.

But the language I quoted above reads as if it goes quite a lot further than that.

Where precisely did this language come from? A lawyer? The policy of some other org?

**Reply by ebullient** [reply]
I was digesting the policy if a few other orgs (at this point, I will have to dig to find the starting point), but was also trying to make it readable by humans... which, in this case, lead to a loss of precision.

This is all draft (now is perfect time to fix it)

In the case of the Linux Foundation, the antitrust policy covers conflict of interest concerns: https://www.linuxfoundation.org/legal/antitrust-policy

Gnome Foundation uses this: https://wiki.gnome.org/Foundation/ConflictOfInterest

**Reply by ebullient** [reply]
Oh. Also this one: https://www.flsenate.gov/Laws/Statutes/2020/0617.0832

**Reply by gavinking** [reply]
The Gnome foundation one looks pretty reasonable I guess.

**Reply by ebullient** [reply]
I'll throw together a PR to revise (based more clearly on Gnome Foundation variant), and we'll see how things go from there.

**Comment by ebullient** [comment]
Sanne, if you're amenable, we can close this discussion, and continue with revisions to https://github.com/commonhaus/foundation-draft/pull/64 ?

**Reply by Sanne** [reply]
Sure, thanks!
